src/egon/data/datasets/demandregio/__init__.py

```diff
@@ -35,10 +38,11 @@ class DemandRegio(Dataset):
     def __init__(self, dependencies):
         super().__init__(
             name="DemandRegio",
-            version="0.0.6",
+            version="0.0.7",
             dependencies=dependencies,
             tasks=(
-                clone_and_install,
+                #clone_and_install, demandregio must be previously installed
```

Comment:
How should it be installed beforehand? Is there any other documentation than this comment?

Reply:
As discussed in our last conversation, we will deal with this in the issue #133